tests/endpoints

```diff
@@ -0,0 +1 @@
+Endpoints: ['tls:127.0.0.1:6262']
```

Comment:
this change on purpose? i expect it is, but figured i'd ask

Reply:
Errant file. Will remove.

Reply:
It looks like a test is creating it...